pkgs/development/compilers/go/1.12.nix

```diff
@@ -31,11 +31,9 @@ stdenv.mkDerivation rec {
   name = "go-${version}";
   version = "1.12";
 
-  src = fetchFromGitHub {
-    owner = "golang";
-    repo = "go";
-    rev = "go${version}";
-    sha256 = "08j7ghcy5cs5p4sw5rqi57lzg52lrix8xbxn87y7y9sv6s3wx44n";
+  src = fetchurl {
+    url = "https://github.com/golang/go/archive/go${version}.tar.gz";
```

Comment:
I would say just use `https://dl.google.com/go/go1.12.src.tar.gz` instead. It is the official source anyway
and release archives should not be regenerated.

Reply:
Good point! I switched it over.